src/test/test_protocol_nmea.cpp

```diff
@@ -25,22 +25,22 @@ TEST(protocol_nmea, nmea_parse_ascb)
     rmci_t rmci[NUM_COM_PORTS] = {};
     int port = 1;
     rmci_t &r = rmci[port];
-    r.periodMultiple[DID_INS_2] = 2;
-    r.periodMultiple[DID_PIMU] = 1;
-    r.periodMultiple[DID_GPS1_POS] = 1;
-    r.nmeaBits = 
+    r.rmcNmea.nmeaPeriod[NMEA_MSG_ID_PINS2] = '1';
```

Comment:
This is curious.... setting the period to the character of the period multiple instead of the value itself?  I'm curious what warranted this change?

Reply:
Easier to debug on failure since the ascii will always be readable in a terminal. No other reason than that. left it because there is no real draw back besides looking weird.